src/components/withLocalize.js

```diff
@@ -130,6 +135,14 @@ class LocaleContextProvider extends React.Component {
         return LocalePhoneNumber.formatPhoneNumber(phoneNumber);
     }
 
+    /**
+     * @param {String} text
+     * @returns {String}
+     */
+    formatPhoneNumberInText(text) {
```

Comment:
I think this function could be moved to ReportTransaction, there is no need to pass it through props